jarbas/api/tests/test_reimbursement_view.py

```diff
@@ -123,76 +89,52 @@ def test_status(self):
-            congressperson_document=2,
-            congressperson_id=1,
-            congressperson_name='Roger That',
-            document_id=42,
-            document_number='6',
-            document_type=7,
-            document_value=8.90,
-            installment=7,
-            issue_date='1970-01-01',
-            leg_of_the_trip='8',
-            month=1,
-            party='Partido',
-            passenger='John Doe',
-            all_reimbursement_numbers=[10, 11],
-            all_reimbursement_values=[12.13, 14.15],
-            all_net_values=[1.99, 2.99],
-            remark_value=1.23,
-            state='UF',
-            subquota_description='Subquota description',
-            subquota_group_description='Subquota group desc',
-            subquota_group_id=5,
-            subquota_id=4,
-            supplier='Acme',
-            term=1970,
-            term_id=3,
-            total_net_value=4.56,
-            total_reimbursement_value=None,
-            year=1970,
-            probability=0.5,
-            suspicions=suspicions,
-            receipt=dict(fetched=False, url=None)
-        )
-        self.assertEqual(expected, contents)
+        for result_attr, result_value in contents.items():
+            if not hasattr(self.reimbursement, result_attr):
+                continue
+            expected_value = getattr(self.reimbursement, result_attr)
+            self.assertEqual(str(result_value), str(expected_value))
 
 
 class TestReceiptApi(TestCase):
 
     def setUp(self):
-        self.obj = Reimbursement.objects.create(**sample_reimbursement_data)
-        self.url = resolve_url('api:reimbursement-receipt', document_id=42)
-        self.expected_receipt_url = 'http://www.camara.gov.br/cota-parlamentar/documentos/publ/13/1970/42.pdf'
+        self.reimbursement = mixer.blend(
+            Reimbursement,
+            year=2017,
+            applicant_id=1,
+            document_id=20,
+            receipt_url='http://www.camara.gov.br/cota-parlamentar/documentos/publ/1/2017/20.pdf'
+        )
+        self.reimbursement_no_receipt = mixer.blend(Reimbursement, receipt_url=None)
+        self.url = resolve_url(
+            'api:reimbursement-receipt', document_id=self.reimbursement.document_id)
+        self.url_no_receipt = resolve_url(
+            'api:reimbursement-receipt', document_id=self.reimbursement_no_receipt.document_id)
 
     @patch('jarbas.core.models.head')
     def test_fetch_existing_receipt(self, mocked_head):
         mocked_head.return_value.status_code = 200
         resp = self.client.get(self.url)
-        expected = dict(url=self.expected_receipt_url)
+        expected = dict(url=self.reimbursement.receipt_url)
         content = loads(resp.content.decode('utf-8'))
         self.assertEqual(expected, content)
 
     @patch('jarbas.core.models.head')
     def test_fetch_non_existing_receipt(self, mocked_head):
         mocked_head.return_value.status_code = 404
-        cache.clear()
```

Comment:
I had understood it was only preparing test to fetch an exiting receipt from a reimbursement without a receipt. So I did the proper preparation instead of clearing cache, what seemed to be an workaround. Since now is easy and simple create new objects I did it. 

Was there any other reason to clear the cache?